SU-SWS/decanter
# 151 footer

# READY FOR REVIEW

# Summary
- Adds new Ligature Font
- Adds new ligature-logo mixin
- Adds new global footer organism (scss & twig)
- Adds new centered-grid-container mixin for centering an item with css grid
- Adds new grid-media mixin for fast creating grids at different breakpoints
- Adds new mixin section in styleguide called "Icons and Logos"
- Fixed up the twig conventions on `{{ attributes }}` and `{{ modifier_class }}`
- Updated the namespace to use the new core directory with twig includes
- Adds new logo twig template (atom)

The "sticky" part of this footer has been intentionally left out of this pull request.

# Steps to Test

1. Check out this branch
2. Fire up your web browser and have a look at the global footer in full page view (http://decanter.sws/item-organisms-global-footer.html)
3. Visually review for aesthetics. 
4. Review HTML & Twig markup for global-footer
5. Review SCSS markup for global-footer
6. Review the rest of the code in this PR and in particular pay attention to the new grid mixins
7. Make comments on my review comments

# Associated Issues and/or People
- @bixgomez 
- @josephgknox 
- @JBCSU 

# Notes
Some General thoughts and notes with working on this component:

1. Grid system is lacking helpers and requires too much code writing. I found it difficult to create a responsive footer element and ended up writing a couple of mixins to help. We should shore up some of the syntax so that we are writing less straight up code into our component and use common mixins.
2. Components could use some refactoring to tease out common elements and made into smaller molecules and atoms. We have a need for atoms and molecules so that we can put together organisms. 
3. There are differing conventions on the twig templates for attributes and classes. Every block level element should have an `{{ attributes }}` twig variable and include a `{{ modifier_class }}` twig variable in the class list. I have remedied this for now but we should document it somewhere.
